lib/worker.js

```diff
@@ -201,15 +196,17 @@ class AssetComputeWorker {
             }
 
         } catch (err) {
-            console.error(`worker() callback processing failed with error after ${callbackTimer.durationSec()} seconds: ${err.message || err}`);
+            this.timers.processingCallback.stop();
+            console.error(`worker() callback processing failed with error after ${this.timers.processingCallback} seconds: ${err.message || err}`);
```

Comment:
a bit confused what `this.timers.processingCallback` will log...seems like its an timers class instance?

are you missing the `.toString()` at the end?

Reply:
does this work because console.error converts the inline variables to strings before logging? (and you override the toString() function to return a timer value)

Reply:
`console.log` calls `toString` behind the scene, and Alex overloaded `toString`: https://github.com/adobe/asset-compute-sdk/pull/61/files#diff-4c41b384c82b232501dd8416ea986ac3R67

See the code to see what is logged exactly, but it will be a duration.

Reply:
Yes. Both `console.log()` and template strings (`${...}`) used here will use `toString()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString#Examples